app/services/db/auth_app_configuration/find_user_by_otp_secret_key.rb

```diff
@@ -0,0 +1,10 @@
+module Db
```

Comment:
Not sure if this is needed

Reply:
Yeah, I'm not sure I see a use for this query